nixos/modules/services/hardware/lirc.nix

```diff
@@ -49,8 +56,7 @@ in {
       description = "LIRC daemon socket";
       wantedBy = [ "sockets.target" ];
       socketConfig = {
-        # default search path
-        ListenStream = "/run/lirc/lircd";
+        ListenStream = cfg.socket;
```

Comment:
Unless we really need this option, I would rather hard-code it.

Reply:
I agree. I need that option to allow other modules like the vdr module to have access to the socket path. That option gives clear dependencies between two modules.